example/todo_list.cpp

```
@@ -36,16 +35,50 @@ class TodoItemData {
-    // data.addTodoItem(
-    //     "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-    //     "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa");
+    data.addTodoItem(
+        "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+        "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa");
     data.addTodoItem("Buy more coffee and monster");
     data.addTodoItem("Finish C++ project");
     data.addTodoItem("Call dad");
   }
 
+  static std::shared_ptr<UIComponent> bodyy() {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)